src/panel/direction/direction_panel.js

```diff
@@ -130,6 +137,10 @@ export default class DirectionPanel {
     PanelManager.resetLayout()
   }
 
+  back() {
+    this.activePanel.closeAction()
```

Comment:
Registering a close event/cb would simplyfy the logic and would avoid the need to update this.activePanel